datasets/huishoudelijkafval/huishoudelijkafval.json

```diff
@@ -185,7 +185,75 @@
             "relation": "bag:nummeraanduidingen",
             "uri": "https://bag.basisregistraties.overheid.nl/def/bag#identificatiecode",
             "description": "Identificatie nummeraanduiding"
-          }
+          },
+          "containerRalKleurNaam": {
+            "provenance": "container_ral_kleur_naam",
+            "type": "string",
+            "description": "De naam van de ralkelur"
+          },
+          "containerRalKleurCode": {
+            "provenance": "container_ral_kleur_code",
+            "type": "string",
+            "description": "De internationale ralkleurcode."
+          },
+          "containerRalKleurHexcode": {
+            "provenance": "container_ral_kleur_hexcode",
+            "type": "string",
+            "description": "De hexcode van de ralkleur."
+          },
+          "containerChipNummber": {
+            "provenance": "container_chip_nummber",
+            "type": "string",
+            "description": "Het identificatienummer van de chip die registreert wanneer de container wordt geleegd."
+          },
+          "containerUnitCardLezerId": {
+            "provenance": "container_unit_card_lezer_id",
+            "type": "string",
+            "description": "Het identificatienummer van de kaartlezer."
+          },
+          "containerKleur": {
+            "provenance": "container_kleur",
+            "type": "string",
+            "description": "De naam van de kleur"
+          },
+          "containerMark": {
+            "provenance": "container_mark",
+            "type": "integer",
+            "description": ""
+          },
+          "containerDatumVervanging": {
+            "provenance": "container_datum_vervanging",
+            "type": "string",
+            "format": "date",
+            "description": "De datum waarop de container wordt vervangen."
+          },
+          "containerDatumWijziging": {
```

Comment:
Als dit een `date-time` is, is het wellicht handiger er `containerTijdstipWijziging` oid. van te maken?

Reply:
inderdaad de datatype is in de database timestamp. ALs we dit aanpassen, moeten we waarschijnlijk alles wat timestamp moeten aanpassen. Het is iets logischer maar ik zal in het team voorleggen.